modules/sdk/project-templates/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/views/greeting.html

```diff
@@ -1,5 +1,6 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Seiphon The Thymeleaf view should not be part of this fix, because Thymeleaf escapes output by default. The `fn:escapeXml` EL expression only works in JSP, since it is tied to JSTL. Please do not include greeting.html in this pull request.

cc @petershin

Reply:
OK, got it, thanks @ngriffin7a